packages/components/src/Offer/Form/Bid.tsx

```diff
@@ -310,6 +312,19 @@ const OfferFormBid: FC<Props> = (props) => {
                 placeholder={t('offer.form.bid.quantity.placeholder')}
                 {...register('quantity', {
                   required: t('offer.form.bid.validation.required'),
+                  validate: (value) => {
+                    if (
+                      parseInt(value, 10) < 1 ||
+                      parseInt(value, 10) > parseInt(props.supply, 10)
+                    ) {
+                      return t('offer.form.bid.validation.in-range', {
+                        quantityAvailable: parseInt(props.supply, 10),
```

Comment:
This one seems wrong also. Can you settle on one name for this variable and use it everywhere.
Maybe something like `max`?

Reply:
normalized all translations